OloEngine/tests/Lua/LuaBindingTest.cpp

```
@@ -0,0 +1,1709 @@
+// =============================================================================
+// Lua Binding Test Fixture
+// =============================================================================
+// Mirrors the bindings from LuaScriptGlue::RegisterAllTypes() for
+// component property round-trip verification. No rendering or Mono required.
+
+class LuaBindingTest : public ::testing::Test
+{
+  protected:
+    sol::state lua;
+
+    void SetUp() override
```

Clang (14.0.6)

[warning] method 'SetUp' can be made static

(readability-convert-member-functions-to-static)